src/cc/bcc_syms.cc

```diff
@@ -108,11 +111,22 @@ bool ProcSyms::resolve_addr(uint64_t addr, struct bcc_symbol *sym) {
 
   sym->module = nullptr;
   sym->name = nullptr;
+  sym->demangle_name = nullptr;
   sym->offset = 0x0;
 
   for (Module &mod : modules_) {
-    if (addr >= mod.start_ && addr < mod.end_)
-      return mod.find_addr(addr, sym);
+    if (addr >= mod.start_ && addr < mod.end_) {
+      bool res = mod.find_addr(addr, sym);
+      if (sym->name) {
+        sym->demangle_name = abi::__cxa_demangle(
+          sym->name, nullptr, nullptr, nullptr
```

Comment:
the coding style looks a bit odd... why not in the same line? other than that it looks great!